Formula/frugal.rb

```diff
@@ -0,0 +1,27 @@
+class Frugal < Formula
+  desc "Thrift on steroids."
+  homepage "https://github.com/Workiva/frugal"
+  url "https://github.com/Workiva/frugal/archive/2.8.1.tar.gz"
+  sha256 "7bcbecee6cc908ed58f3dc7963b101eda43cbe9ab139d655d8c4e0dd670f06ce"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/Workiva/frugal"
+    dir.install buildpath.children
+    cd dir do
+      system "godep", "restore"
+      system "go", "build", "-o", bin/"frugal"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    (testpath/"test.frugal").write("typedef double Test")
+
+    # Generate the most basic of python files to verify install.
+    system "#{bin}/frugal", "--gen", "py", testpath/"test.frugal"
```

Comment:
Does this generate a specific file with certain contents? We should verify it exists and has the correct contents if so.